cmake/sample_defs/cpu1_cfe_es_startup.scr

```diff
@@ -4,6 +4,7 @@ CFE_APP, sample_app,  SAMPLE_APP_Main,    SAMPLE_APP,   50,   16384, 0x0, 0;
 CFE_APP, ci_lab,      CI_Lab_AppMain,     CI_LAB_APP,   60,   16384, 0x0, 0;
 CFE_APP, to_lab,      TO_Lab_AppMain,     TO_LAB_APP,   70,   16384, 0x0, 0;
 CFE_APP, sch_lab,     SCH_Lab_AppMain,    SCH_LAB_APP,  80,   16384, 0x0, 0;
+CFE_APP, cfe_testcase, CFE_TestMain,   CFE_TEST_APP, 100,   16384, 0x0, 0;
```

Comment:
Preference is to start the cfe_testcase app via command instead of editing the default startup script.